src/documentation/hello-claims.md

```diff
@@ -23,6 +23,14 @@ Following are the scopes currently supported by Hellō. These are standard OpenI
 | `ethereum`|A verified ethereum address.|
 |`profile_update`|The user will be prompted to release updated profile information.<br> [See FAQ 14](/faqs/#_14-how-can-users-update-the-profile-information-i-received-from-hello-for-example-the-user-wants-to-change-their-profile-picture-or-email) for details.|
 
+## Profile Update
+You can update existing claims by including the `profile_update` scope in your authorization request.
+The `profile_update` scope can be utilized under the following conditions:
+- You must already have one of the following scopes: `email`, `phone`, `picture`, or `ethereum` to update.
```

Comment:
line 29 and 30 are almost the same thing. 

rework of 27 -- 30
You can update existing claims by including the `profile_update` scope in your authorization request. 
- You can only update a claim you have already received
- You can only update `email`, `phone`, `picture`, or `ethereum`